molo/core/tests/test_search.py

```diff
@@ -106,6 +106,27 @@ def test_search(self):
         response = self.client.get(reverse('search'))
         self.assertContains(response, 'No search results for None')
 
+    def test_search_empty_values(self):
+        self.backend = get_search_backend('default')
+        self.backend.reset_index()
+
+        self.mk_article(
+            self.english_section, title="Site 1 article")
+        self.mk_article(
+            self.yourmind2, title="Site 2 article")
+        self.backend.refresh_index()
+
+        response = self.client.get(reverse('search'), {
+            'q': ' '
+        })
+        self.assertEqual(response.status_code, 200)
```

Comment:
Is this page expected to contain "No search results for <query>" ?

Reply:
well that is how the behavior is, its safe to assume that is the intended outcome... makes no search in searching for an empty string.